packit_service/worker/jobs.py

```diff
@@ -818,7 +818,7 @@ def push_statuses_metrics(
             # we need more info why this has happened
             logger.debug(f"Event dict: {self.event}.")
             logger.error(
-                f"Event {self.event.__class__.__name__} took ({response_time}s) to process."
+                f"Event {self.event.__class__.__name__} took more than 15s to process."
```

Comment:
~~Is there any way we could get the response times in?~~ Though it looks like we have `pushgateway.last_initial_status_time` below, so it should be possible to run some stats on it.